activerecord/test/schema/schema.rb

```diff
@@ -370,6 +374,11 @@ def except(adapter_names_to_exclude)
     t.integer :ideal_reference_id
   end
 
+  create_table :joke_comments, force: true do |t|
```

Comment:
is it necessary to create new tables to test this feature? Can't we reuse some tables that already exist? We try to not add more global models for single test scenarios like this.